Makefile

```diff
@@ -157,7 +157,7 @@ run:
 	@quickfeed -service.url $(DOMAIN) -database.file ./tmp.db
 
 runlocal:
-	@quickfeed -service.url 127.0.0.1
+	@quickfeed -dev -service.url 127.0.0.1:8081
```

Comment:
Should we use the `-dev` only shorthand? Or do we want the `-service.url` also?

Reply:
We need it because the default `service.url` is empty, but it is used to construct the callback and webhook URLs so we need it to be correct. 

One option is to add the logic that will set `127.0.0.1` in combination with a correct port as `baseURL` on start in case `-dev` is used.

Reply:
The reason I mentioned this was this code:
```go
	if *dev {
		*baseURL = "127.0.0.1" + *httpAddr
	}
```
Which seems to make the `-service.url 127.0.0.1:8081` unnecessary when you use `-dev`

Reply:
Would it be a possibility to remove `-service.url` and use `*baseURL = "127.0.0.1" + *httpAddr` (or something like  `127.0.0.1 + env.Port()`) if `-dev`, and `env.Domain()` otherwise?

Reply:
> Would it be a possibility to remove `-service.url` and use `*baseURL = "127.0.0.1" + *httpAddr` (or something like `127.0.0.1 + env.Port()`) if `-dev`, and `env.Domain()` otherwise?

I like this idea.